crates/optimism/primitives/src/lib.rs

```diff
@@ -9,3 +9,172 @@
 
 pub mod bedrock;
 pub mod op_tx_type;
+
+use alloy_primitives::{bytes, Bytes, TxKind, Uint, B256};
+
+use alloy_consensus::{SignableTransaction, TxLegacy};
+use alloy_rlp::{Decodable, Encodable, Result};
+use op_tx_type::OpTxType;
+use reth_primitives::revm_primitives::{AccessList, SignedAuthorization};
+use reth_primitives_traits::{InMemorySize, TransactionExt};
+
+use op_alloy_consensus::{OpTxEnvelope, OpTypedTransaction};
+
+#[cfg(feature = "arbitrary")]
+use arbitrary::Arbitrary;
+
+#[cfg_attr(feature = "arbitrary", derive(Arbitrary))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+/// Optimistic transaction type.
```

Comment:
```suggestion
/// Optimistic transaction.
```
Nitpick, can get a bit confusing with the transaction envelope type id, often referred to as 'transaction type'